backend/codingchallenge/api/views/applications.py

```diff
@@ -0,0 +1,105 @@
+import time
+import random
+
+# Authentication imports
+from rest_framework.permissions import IsAdminUser, IsAuthenticated
+
+# RESTapi imports
+from rest_framework import status
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+# import model
+from ..models import Application, Challenge
+
+# import serializer
+from ..serializers import ApplicationSerializer
+
+# import errorMessage class
+from . import errorMessage
+
+### endpoint: /api/admin/applications
+class AdminApplicationsView(APIView):
+    # grant permission only for admin user
+    permission_classes = [IsAdminUser]
+
+    name = "Admin Application View"
+    description = "handling all requests for applications as a admin"
+
+    # 1. List all
+    def get(self, request, *args, **kwargs):
+        '''
+        get Application with follwing columns:
+            applicationId,
+            operatingSystem,
+            programingLanguage,
+            expiry,
+            submission,
+            githubRepo,
+            status,
+            created,
+            modified
+        '''
+        applications = Application.objects
```

Comment:
According to the django documentation you should use Application.objects.all() to query all objects.

Reply:
will be implemented by #61. Thats not our Baustelle